charts/dave/charts/document-storage/templates/service.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "daveDocumentStorage.fullname" . }}
+  labels:
+    {{- include "daveDocumentStorage.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)